composer.json

```
@@ -52,6 +52,7 @@
 		"helmich/typo3-typoscript-lint": "^3.1.1",
 		"icanhazstring/composer-unused": "^0.8.11",
 		"php-coveralls/php-coveralls": "^2.7.0",
+		"phpmd/phpmd": "^2.15",
```

For development dependencies, please let's always use the exact version:

```suggestion
		"phpmd/phpmd": "^2.15.0",
```